src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -107,6 +107,13 @@ public class WalletHistoryRequest : RequestModel
         public string WalletName { get; set; }
 
         public string AccountName { get; set; }
+
+        public int? Skip { get; set; }
```

Comment:
Cant you just default this to 0 then you don't have to do a coalesce?

Reply:
For Skip that wouldn't be a problem, but then how do I interpret `Take = 0`?
If Take is 0 I should normally return no items, or write some code that says that if it's 0 then it's means it's off.
To avoid all confusion, a nullable is the easiest way.